packages/tests-unit/tests/core/routing/matcher.test.ts

```diff
@@ -490,6 +490,21 @@ describe("fixDataPage", () => {
     expect(response).toEqual(event);
   });
 
+  it("should not return 404 for data requests (with base path) that don't match the buildId", () => {
+    NextConfig.basePath = '/base';
+
+    const event = createEvent({
+      url: "/base/_next/data/abc/test",
+    });
+
+    const response = fixDataPage(event, "abc");
+
+    expect(response.statusCode).not.toEqual(404);
+    expect(response).toEqual(event);
+
+    NextConfig.basePath = undefined;
```

Comment:
Don't need this one as `NextConfig` gets reset before each : https://github.com/opennextjs/opennextjs-aws/blob/a322aa699ac101a9bb46840c45edeefa6b661bb5/packages/tests-unit/tests/core/routing/matcher.test.ts#L39